solutions/prob45.js

```diff
@@ -0,0 +1,18 @@
+const Kebab =(a,b='',i=0)=>{
```

Comment:
Good job, you're almost there! I tested your code but it didn't return the kebab case. The kebab case should be:
 input: 'Maricris Bonzo' & Output: 'maricris-bonzo'. Please try again! & also, please create a test file for this.<img width="442" alt="screen shot 2017-10-27 at 9 54 23 pm" src="https://user-images.githubusercontent.com/11692241/32131384-9797faf2-bb61-11e7-8f92-6e041e9c9e79.png">